GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -0,0 +1,431 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import AppAuthCore
+import GTMSessionFetcherCore
+
+/// An implementation of the @c GTMFetcherAuthorizationProtocol protocol for the AppAuth library.
+///
+/// Enables you to use AppAuth with the GTM Session Fetcher library.
+@objc open class GTMAppAuthFetcherAuthorization: NSObject,
+                                                 GTMFetcherAuthorizationProtocol,
+                                                 NSSecureCoding {
+  /// The AppAuth authentication state.
+  @objc public let authState: OIDAuthState
```

Comment:
Add an empty line after each of these documented class properties to increase readability?